test/functional/helpers.lua

```diff
@@ -302,6 +302,10 @@ end
 local function nvim_feed(input)
   while #input > 0 do
     local written = module.request('nvim_input', input)
+    if written == nil then
+      module.assert_alive()
```

Comment:
`module.assert_alive` does not call `assert`, but returns something only.

Reply:
`eq()` does an assert. But `return` can be removed.